ui/src/pages/ShoeMap.tsx

```diff
@@ -1,11 +1,37 @@
 import * as React from "react";
+import styled from "styled-components";
+import { Map, TileLayer } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+
 import { TitleText } from "../styles/typography";
 
+const StyledMap = styled(Map)`
+  height: 70vh;
+  width: 90vw;
+  display: flex;
+`;
+
+const MapContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
 export const ShoeMap: React.FC = () => {
+  const zoom = 12;
+  const currentLocation = { lat: 43.4723, lng: -80.5449 };
```

Comment:
Should these be props? If we are going to reuse this component for maps in the stories that could be helpful but if not it is probably fine like this.

Reply:
Yeah for sure, I just hardcoded the values temporarily. I think I'll hold off on adding them to the props until we have a clearer idea of what we're building